src/features/notifications/AccountSlider.tsx

```diff
@@ -62,8 +63,9 @@ const AccountSlider = () => {
       // clear the opened push notification
       setOpenedNotification(undefined)
     } else {
+      // TODO: Double check with team if this is the desired behavior
       // set the selected list as the first account by default, after sorting of data by unread
-      await updateSelectedList(data[0])
+      // await updateSelectedList(data[0])
```

Comment:
What's going on here? This seems fine? I'll test it out in a bit.

Reply:
Yeah it was causing an infinite render when inside the header component and wasn't even sure if we needed this